checks.d/docker_daemon.py

```diff
@@ -307,6 +311,16 @@ def _get_and_count_containers(self):
 
             containers_by_id[container['Id']] = container
 
+            # grab pid via API if custom cgroups - otherwise we won't process find when
+            # crawling for pids.
+            if custom_cgroups:
+                try:
+                    inspect_dict = self.docker_client.inspect_container(container_name)
```

Comment:
This call can be very slow sometimes.
We'll need to advise our users to use this feature as little as possible (preferably as a comment in the yaml file). Did you have a chance to try this on a lot of containers? I wonder if the collector run could get timeout killed because of this.